qiita_pet/test/test_study_handlers.py

```diff
@@ -12,6 +12,37 @@
 class TestHelpers(TestHandlerBase):
     database = True
 
+    def setUp(self):
+        self.exp = [{
+            'status': 'public',
+            'checkbox': "<input type='checkbox' value='1' />",
+            'abstract': 'This is a preliminary study to examine the microbiota'
+            ' associated with the Cannabis plant. Soils samples from the bulk '
```

Comment:
Aligning the quotes will improve the ability to differentiate between keys and values, for instance:

```python
self.exp = [{
    ...
    'abstract': 'foo bar '
                'more and more and more text '
                'even more text',
    'owner': 'stuff',
    ...
}]
```